apm-sync.sh

```diff
@@ -4,13 +4,34 @@ set -e
 set -u
 
 
+function is_darwin() {
+	if [ "$(uname | lower)" == 'darwin' ]; then
+		exit 0
+	else
+		exit 1
+	fi
+}
+
+
+function get_col() {
+
+	## Package names are printed in the third column only in darwin(osx)
+	## https://github.com/atom/apm/blob/master/src/stars.coffee#L80
+	if [ is_darwin ]; then
```

Comment:
No need to use '[ ]' when checking an exit status of a command/function.
In fact, the '[' (test command) in this case will always return true irrespective of the platform.

```suggestion
	if is_darwin; then
```